### PR TITLE
Refine errors for import

### PR DESCRIPTION
**Pull request by yah01**
/kind improvement

### CODE REVIEW

File: internal/proxy/impl.go [hunk 1]
```diff
@@ -1615,9 +1615,9 @@ func (node *Proxy) GetLoadState(ctx context.Context, request *milvuspb.GetLoadSt
 	var progress int64
 	if len(request.GetPartitionNames()) == 0 {
 		if progress, _, err = getCollectionProgress(ctx, node.queryCoord, request.GetBase(), collectionID); err != nil {
-			if errors.Is(err, ErrInsufficientMemory) {
+			if errors.Is(err, merr.ErrServiceMemoryLimitExceeded) {
 				return &milvuspb.GetLoadStateResponse{
-					Status: InSufficientMemoryStatus(request.GetCollectionName()),
+					Status: merr.Status(err),
 				}, nil
 			}
```
Comment (congqixia) [inline internal/proxy/impl.go:1622]: is grpc error handling missing here?

File: internal/rootcoord/import_manager.go [hunk 2]
```diff
@@ -419,21 +419,16 @@ func (m *importManager) isRowbased(files []string) (bool, error) {
 // importJob processes the import request, generates import tasks, sends these tasks to DataCoord, and returns
 // immediately.
 func (m *importManager) importJob(ctx context.Context, req *milvuspb.ImportRequest, cID int64, pID int64) *milvuspb.ImportResponse {
-	returnErrorFunc := func(reason string) *milvuspb.ImportResponse {
+	if req == nil || len(req.Files) == 0 {
```
Comment (congqixia) [inline internal/rootcoord/import_manager.go:422]: NIT: 
```suggestion
	len(req.GetFiles()) == 0
```

File: internal/rootcoord/import_manager.go [hunk 3]
```diff
@@ -786,24 +780,24 @@ func (m *importManager) getTaskState(tID int64) *milvuspb.GetImportStateResponse
...
+	v, err := m.taskStore.Load(BuildImportTaskKey(tID))
+	if err != nil || v == "" {
 		log.Warn("failed to load task info from Etcd",
 			zap.String("value", v),
-			zap.Error(err))
+			zap.Error(err),
+		)
+		resp.Status = merr.Status(err)
+		return resp
 	}
```
Comment (congqixia) [inline internal/rootcoord/import_manager.go:791]: `v==""` returns success response here, is that OK?

**Review by jiaoew1991 — APPROVED** [review]
/lgtm

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**